README.md

```diff
@@ -1004,17 +1004,18 @@ If View State is sliced, not given or `{}`, all not-mentioned properties retain
 
 Translation keys convention:
 
-Translation Target | Translation Key
----|---
-Model name (shown in the header) | `model.name`
-Model tab label | `model.tab.<tab name>.label`
-Model section label | `model.section.<section name>.label`
-Model field label | `model.field.<field name>.label`
-Model field hint | `model.field.<field name>.hint`
-Model field tooltip | `model.field.<field name>.tooltip`
-Custom [Field/Instance Validation Error](#parsing-error-and-fieldinstance-validation-error) message | `model.field.<field name>.error.<error id>`
-
-If some translation is not provided, the corresponding label/message text is obtained by converting camelcase id/name to titlecase. For example, `maxOrderValue` is displayed as `Max Order Value`.
+Translation Target | Translation Key | Default translation
+---|---|---
+Model name (shown in the header) | `"model.name"` | `"model.name"`
+Model tab label | `"model.tab.<tab name>.label"` | `titleCase("<tab name>")`
+Model section label | `"model.section.<section name>.label"` | `titleCase("<section name>")`
+Model field label | `"model.field.<field name>.label"` | `titleCase("<field name>")`
+Model field hint | `"model.field.<field name>.hint"` | -
+Model field tooltip | `"model.field.<field name>.tooltip"` | -
+Custom [Field Validation Error](#parsing-error-and-fieldinstance-validation-error) | `"model.field.<field name>.error.<error id>"` | `error.message \|\| error.id`
+[Instance Validation Error](#parsing-error-and-fieldinstance-validation-error) | `"model.error.<error id>"` | `error.message`
```

Comment:
`error.message` is optional according to [documentation](https://github.com/OpusCapita/react-crudeditor/blob/issue-217/README.md#parsing-error-and-fieldinstance-validation-error). Stated default also contradicts [actual implementation](https://github.com/OpusCapita/react-crudeditor/blob/issue-217/src/crudeditor-lib/middleware/notifications.js#L123).